pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2532,9 +2549,37 @@
 ::  +load: migrate old state to new state (called on vane reload)
 ::
 ++  load
-  |=  old=axle
+  =>  |%
+    ++  axle-old
+      %+  cork
+        axle
+      |=  =axle
+      axle(date %~2020.10.18)
+  --
+  |=  old=$%(axle axle-old)
   ^+  ..^$
-  ..^$(ax old)
+  ::
+  ~!  %loading
+  ?-    -.old
+      %~2020.10.18
+    =/  [* d=duct *]
+      %+  snag  0
+      %+  skim  bindings.server-state.old
+        |=  [=binding =duct =action]
+        =(%authentication -.action)
+    =/  new
+    %=  old
+      date  %~2022.7.26
+    ::
+      bindings.server-state
+      ^-  (list [binding duct action])
+      :-  [[~ /~/name] d [%name ~]]  bindings.server-state.old
```

Comment:
You'll want to call `+insert-binding`, the ordering in this list does actually matter.

```suggestion
        bindings.server-state
      (insert-binding [[~ /~/name] d [%name ~]] bindings.server-state.old)
```

Reply:
This whole block in/under the `=/  new` could use some indents or further styling love. You can probably get away with just `%=`ing right away, something like:

```hoon
%=  $
  date.old  %~2022.7.26
  bindings.server-state.old  etc
==
```

Reply:
Fixed.